tests/test-extensions/test-oauth-extension/src/main/java/org/mule/test/oauth/ClientCredentialsConfig.java

```diff
@@ -6,13 +6,24 @@
  */
 package org.mule.test.oauth;
 
+import org.mule.runtime.api.lifecycle.Disposable;
 import org.mule.runtime.extension.api.annotation.Configuration;
 import org.mule.runtime.extension.api.annotation.Operations;
 import org.mule.runtime.extension.api.annotation.connectivity.ConnectionProviders;
 
 @ConnectionProviders(TestOAuthClientCredentialsProvider.class)
 @Operations(TestOAuthOperations.class)
 @Configuration(name = "client-credentials")
-public class ClientCredentialsConfig {
+public class ClientCredentialsConfig implements Disposable {
```

Comment:
where is the test case for this?

Reply:
see https://github.com/mulesoft/mule-integration-tests/pull/1332/files